src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+$days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6
+);
+$calendar-font: Arial, Helvetica, sans-serif;
+$days-in-row: 7;
+$square-size: 100px;
+$square-gap-size: 1px;
+$square-size-full: calc($square-size + $square-gap-size);
+$calendar-width: calc($square-size-full * $days-in-row);
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  font-family: $calendar-font;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-gap-size;
+  padding: 10px;
+  width: $calendar-width;
+
+    @each $day, $day-number in $days {
+      &--start-day-#{$day} &__day:first-child {
+        margin-left: $day-number * $square-size-full;
+      }
+    }
+
+    @for $days-number from 28 through 31 {
+      &--month-length-#{$days-number} &__day:nth-child(n + #{$days-number + 1}) {
+        display: none;
+      }
+    }
+
+&__day {
```

Comment:
use as nesting at `calendar`

Reply:
> use as nesting at `calendar`

it's like, like i already did calendar  (calendar__day) &__day  and (caldendar_day hover ) &day @Viktor-Kost